packages/owl-bot/test/handlers.ts

```diff
@@ -113,12 +116,16 @@ describe('handlers', () => {
       };
       // Mock the database helpers used to check for/update existing PRs:
       class FakeConfigStore implements ConfigsStore {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
         getConfigs(repo: string): Promise<Configs | undefined> {
           throw new Error('Method not implemented.');
         }
         storeConfigs(
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
           repo: string,
```

Comment:
mocking whole interfaces creates several unused variables, it would be good if our linter allowed `_foo` in tests.